src/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -173,7 +173,7 @@ private PendingIntent getMarkAsReadIntent(int chatId, boolean markNoticed) {
 
     // channelIds: CH_MSG_* are used here, the other ones from outside (defined here to have some overview)
     public static final String CH_MSG_PREFIX = "ch_msg";
-    public static final String CH_MSG_VERSION = "4";
+    public static final String CH_MSG_VERSION = "5";
```

Comment:
this is needed to avoid reusing old deleted channels with wrong configuration (default sound in silent mode)